libs/database/src/collab/collab_db_ops.rs

```diff
@@ -194,27 +199,56 @@ pub async fn insert_into_af_collab_bulk_for_user(
   Ok(())
 }
 
+#[inline]
+pub async fn select_collabs_created_since<'a, E>(
+  conn: E,
+  workspace_id: &Uuid,
+  since: DateTime<Utc>,
+) -> Result<Vec<AFCollabData>, sqlx::Error>
+where
+  E: Executor<'a, Database = Postgres>,
+{
+  let records = sqlx::query_as!(
+    AFCollabData,
+    r#"
+        SELECT c.oid, c.partition_key, c.updated_at, c.blob
+        FROM af_collab c
+        WHERE c.workspace_id = $1
+            AND c.deleted_at IS NULL
+            AND c.created_at > $2
+        ORDER BY updated_at
+        LIMIT 500
```

Comment:
Why are we selecting 500? Shouldn’t we use the value passed in by the caller? What happens if those 500 rows are actual database records?

Reply:
I can change it into a variable, but the problem stays - we don't want to overdo a number of collabs to be returned, since we could easily flood the user.